testing/tests/DevExpress.ui.widgets.editors/colorBox.tests.js

```diff
@@ -578,6 +578,22 @@ QUnit.module('Color Box', {
         colorBox.open();
         assert.equal(colorBox._colorView.option('stylingMode'), 'underlined');
     });
+
+    [
+        { value: undefined, editAlphaChannel: false },
+        { value: undefined, editAlphaChannel: true },
+        { value: null, editAlphaChannel: false },
+        { value: null, editAlphaChannel: true }
+    ].forEach(({ value, editAlphaChannel }) => {
+        QUnit.test(`Text should be empty when value=${value} and editAlphaChannel=${editAlphaChannel}`, function(assert) {
+            const $colorBox = $('#color-box').dxColorBox({
+                value,
+                editAlphaChannel
+            }); const colorBox = $colorBox.dxColorBox('instance');
+
+            assert.notOk(colorBox.option('text'));
```

Comment:
Will be better to do `strictEqual` for this expected value. It is not clear to me that means notOk - '', 0, 'undefined', null